_global/css/adversary.css

```diff
@@ -195,6 +214,19 @@ icon.city{
   height:19px;
   width:19px;
   margin-top:-5px;
+  background-image: url('../images/50px-Citiyicon.png');
+}
+
+icon.escalation{
+  height:17px;
+  width:15px;
+  margin-bottom:-2px;
+  margin-top:-10px;
+  background-image: url('../images/25px-Stage_II_Escalation.png');
+}
+
+icon.presence{
+  background-image: url('../images/board/50px_presence.png');
```

Comment:
I updated some of the blurry icons to use higher resolution icons. I think all of these already exist in the master repo, but I'll have to double check. If the icons disappear when this is merged, that's why.

Reply:
Let's check this before merging.

Reply:
I've added some icons since - simple elements and other things. Thoughts/concerns? Is file size an issue? image folder is 15MB